azure/storage.sh

```diff
@@ -1,14 +1,15 @@
 # Storage related functions
 
 fn azure_storage_account_create(name, group, location, sku, kind) {
```

Comment:
I think we must document the function saying that `name` will be used as suffix of a random name. And maybe point the user to azure documentation regarding storage locality.